### PR TITLE
Conda appveyor master

### PR DESCRIPTION
**Pull request by TomAugspurger**
xref https://github.com/conda/conda/issues/6251

Closes https://github.com/pandas-dev/pandas/issues/18073

### CODE REVIEW

**Comment by TomAugspurger** [comment]
That apparently wasn't it. (We'll still want to update to the newer miniconda though).

File: ci/requirements-2.7_WIN.run [hunk 1]
```diff
@@ -16,3 +16,4 @@ bottleneck
 html5lib
 beautifulsoup4
 jinja2=2.8
+moto
```
Comment (gfyoung) [inline ci/requirements-2.7_WIN.run]: Newline at end of file.  Same for the 3.6 version.

**Comment by jreback** [comment]
> That apparently wasn't it. (We'll still want to update to the newer miniconda though).

we already use the latest windows installeer, e.g. see ``ci/install.ps1``

File: appveyor.yml [hunk 2]
```diff
@@ -18,13 +18,13 @@ environment:
 
   matrix:
 
-    - CONDA_ROOT: "C:\\Miniconda3_64"
+    - CONDA_ROOT: "C:\\Miniconda36_64"
```
Comment (jreback) [inline appveyor.yml]: this doesn't actually do anything other than create a directory for us to *install* into

**Comment by jreback** [comment]
superseded by #18105

**Pull request closed without merging**